packages/keyutils/keyutils.spec

```diff
@@ -0,0 +1,85 @@
+Name: %{_cross_os}keyutils
+Version: 1.6.1
+Release: 1%{?dist}
+Summary: Linux Key Management Utilities
+License: GPLv2+ and LGPLv2+
+Url: http://people.redhat.com/~dhowells/keyutils/
+Source0: http://people.redhat.com/~dhowells/keyutils/keyutils-%{version}.tar.bz2
+Source1: keyutils-tmpfiles.conf
+Source2: request-key.conf
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Development package for building Linux key management utilities
+
+%description devel
+%{summary}.
+
+%prep
+%setup -n keyutils-%{version} -q
+
+%global set_env \
+%set_cross_build_flags \\\
+export CC=%{_cross_target}-gcc \\\
+%{nil}
+
+%global keyutilsmake \
+%make_build \\\
+  NO_ARLIB=1 \\\
+  ETCDIR=%{_cross_sysconfdir} \\\
+  LIBDIR=%{_cross_libdir} \\\
+  USRLIBDIR=%{_cross_libdir} \\\
+  BINDIR=%{_cross_bindir} \\\
+  SBINDIR=%{_cross_sbindir} \\\
+  MANDIR=%{_cross_mandir} \\\
+  INCLUDEDIR=%{_cross_includedir} \\\
+  SHAREDIR=%{_cross_datadir}/keyutils \\\
+%{nil}
+
+
+%build
+%set_env
+%keyutilsmake \
+  RELEASE=.%{release} \
+  NO_GLIBC_KEYERR=1 \
+  CC="${CC}" \
+  CFLAGS="${CFLAGS}" \
+  LDFLAGS="${LDFLAGS}"
```

Comment:
nit: you could probably move all this into `%keyutilsmake` also

Depending on how the Makefile is set up, changes to some of these variables - which happens implicitly when omitting them when they were previously defined - can cause `make install` to actually rebuild the binaries, potentially in an incorrect way if the compiler changes.

Here I don't think it matters, but it makes me uncomfortable to see the flags changing across `%build` and `%install` because I've been burned by it before.